canonical-websites/www.ubuntu.com
# Add system information legal notice page

## Done

Add system information legal notice page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/legal/system-information>
- Make sure the page matches [the copy doc](https://docs.google.com/document/d/1njLcJ6LToh-t8pekxwibU9r7RTOhbRY0itIIjzeljxw/edit)


## Issue / Card
Fixes #3207
